features/full_tests/batch_1/cached_error_reports.feature

```diff
@@ -2,7 +2,6 @@ Feature: Cached Error Reports
 
 Scenario: If an empty file is in the cache directory then zero requests should be made
     When I run "EmptyReportScenario" and relaunch the app
-    And I configure the app to run in the "non-crashy" state
     And I run "EmptyReportScenario"
```

Comment:
It'd be good to have more of a distinction between the different run steps IMO. I can see developers unfamiliar with mazerunner getting confused about the difference of appending `and relaunch the app` onto the run step.

It also seems to be somewhat inconsistent across the feature files - sometimes it's `I run X`, and at other times it's `I configure X`.

Reply:
I think the inconsistency is actually an omission on my part - it should be using `I configure Bugsnag for X` consistently after a crash.

I'm happy to change the wording of the run steps to aid clarity, it probably just needs a bit of further thought and analysis to make sure we capture the logical situations correctly.  I've taken a note to look at that, in a view to raising a separate ticket for it.

Reply:
Splitting this out sounds sensible.